webpack.common.js

```diff
@@ -6,6 +6,7 @@ const autoprefixer = require( 'autoprefixer' );
 const externals = {
 	jquery: 'jQuery',
 	'@eventespresso/eejs': 'eejs',
+	'@eventespresso/i18n': 'eejs.i18n',
```

Comment:
there's a problem with this entry because there are now two entries for `@eventespresso/i18n`.  This might be a merge artifact but there should only be one.  If you remove this one, it will most closely match what I see in `Gutenberg/master`

Reply:
ya they are identical, will fix